lib/internal/errors.js

```diff
@@ -148,6 +148,7 @@ E('ERR_SOCKET_BAD_TYPE',
 E('ERR_SOCKET_CANNOT_SEND', 'Unable to send data');
 E('ERR_SOCKET_BAD_PORT', 'Port should be > 0 and < 65536');
 E('ERR_SOCKET_DGRAM_NOT_RUNNING', 'Not running');
+E('ERR_INVALID_FD', 'fd must be positive integer');
```

Comment:
To be consistent with the existing error message, something like the following would be appropriate:

```js
E('ERR_INVALID_FD', (fd) => `"fd" must be a positive integer: ${fd}`);
```